### ISSUE TITLE
feature request: move to icloud

### ISSUE THREAD

**Issue by kasperkamperman** [Ranchero-Software/NetNewsWire#4230]
Is there a way to move the "On My iPhone" feeds and starred items to iCloud?
So I can access them as well from the MacOS Desktop application? 

I couldn't find this in documentation (or overlooked it). 

Thanks for this App.

**Comment by xinteksik** [comment]
Yes. Export feeds first, then import feeds to cloud.  Simple workaround.

**Comment by kasperkamperman** [comment]
Thanks. But do I keep my starred items in that way?
